.github/workflows/stale.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           This Pull Request is stale because it has been open for 14 days with
           no activity. It will automatically close after 7 more days of
           inactivity.
-        stale-pr-label: 'kind/stale'
+        stale-pr-label: 'kind/stale,kind/enhancement'
```

Comment:
next line?

Reply:
No, it wants a CSV

Reply:
actually, looking closer, shouldn't this go on line 18?

adding kind/enhancement to the exempt list. Not to the labels to add on to a stale PR

Reply:
Ah yup